docs/Configuration.rst

```diff
@@ -5981,7 +6036,166 @@ data is stored in the database.
 To disable the login history data storage in the database, simply set the parameter 
 ``exo.audit.login.enabled`` to ``false``.
 
+.. _Configuration.Wallet:
+
+======================
+Configuring eXo Wallet
+======================
+
+The eXo Wallet addon uses the `Ethereum Blockchain <https://www.ethereum.org/>`__ to reward users based on a crypto-currency managed inside this blockchain.
+
+Blockchain network settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The addon is configured by default to communicate with Ethereum Mainnet throught `Infura <https://infura.io/>`__ using websocket and http protocols:
+
+.. code-block:: jproperties
+
+    exo.wallet.blockchain.networkId=1
+    exo.wallet.blockchain.network.websocket=wss://mainnet.infura.io/ws/v3/a1ac85aea9ce4be88e9e87dad7c01d40
+    exo.wallet.blockchain.network.http=https://mainnet.infura.io/v3/a1ac85aea9ce4be88e9e87dad7c01d40
+
+- The Websocket endpoint is used to:
+
+   -  Scan new mined blocks on blockchain to verify if there is a new mined transaction that involves a wallet of a recognized user.
+      The periodicity can be configured using the following property (Default value: every hour):
+
+    .. code-block:: jproperties
+
+          exo.wallet.ContractTransactionVerifierJob.expression=0 0 * ? * * *
+
+   -  Checks all pending transactions sent using wallet application to update its state internally in internal store and to send notifications for receiver and sender.
+      The check is made periodically and can be configured using the following property (Default value: every 10 seconds):
+
+    .. code-block:: jproperties
+
+          exo.wallet.PendingTransactionVerifierJob.expression=0/10 * * * * ?
+
+   -  Send transactions on blockchain using *Admin wallet* that is used to administrate all other wallets.
```

Comment:
```suggestion
   -  Send transactions from the so-called *Admin wallet* which is a dedicated blockchain account with special privileges than is used to administrate (initialize, approved, delete, reward, etc.)  all user wallets.
```

Reply:
In fact it initializes and rewards users only

Reply:
done